docker/docker-compose.yml

```diff
@@ -70,7 +70,7 @@ services:
     restart: always
 
   pictrs:
-    image: asonix/pictrs:0.3.1
+    image: asonix/pictrs:0.4.0-beta.17
```

Comment:
@asonix should we be using a beta here? Doesn't matter to me, as long as the upgrade migration runs smoothly, and you don't forsee any breaking changes.

Reply:
@dessalines asonix and I tested this(b16), found a bug and fixed it for b17. Migration with a full db works successfully.

Reply:
The beta should be fine for everything lemmy currently needs. There's two more things I want to change before issuing a release candidate:
- remove 0.4's image caching feature. It's not a good API
- Support transparency in gif, vp8, and vp9